### PR TITLE
Cloud hosted adjustments

### PR DESCRIPTION
**Pull request by AustinSeto**
Adjust some wordings for the cloud hosted version of the guide

### CODE REVIEW

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]
@AustinSeto would you not delete the .travis.yml.disabled for now? We will remove it later.

File: README.adoc [hunk 1]
```diff
@@ -166,9 +166,8 @@ curl -k https://localhost:8443/system/properties/os
 ```
 {: codeblock}
 
-You get a blank screen and aren't given
-access because you didn't supply a valid JWT with the request. The following
-error also appears in the command-line session of the **system** service:
+You get an empty response since you don't have access because you didn't supply a valid JWT with the request.
```
Comment (gkwan-ibm) [inline README.adoc:169]: `since` and `because` in one statement looks not right

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]

File: README.adoc [hunk 2]
```diff
@@ -166,9 +166,9 @@ curl -k https://localhost:8443/system/properties/os
 ```
 {: codeblock}
 
-You get a blank screen and aren't given
-access because you didn't supply a valid JWT with the request. The following
-error also appears in the command-line session of the **system** service:
+There response is empty because you don't have access.
```
Comment (dmuelle) [inline README.adoc:169]: There response is empty
--->
The response is empty

**Review by dmuelle — CHANGES_REQUESTED** [review]
one typo to fix

**Review by dmuelle — APPROVED** [review]
LGTM

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**